galaxy/wrapper/test-data/profiler_result2.tabular

```diff
@@ -0,0 +1,3 @@
+bin labels		-0.0Kb	0.0Kb
```

Comment:
I guess we have three options:
 1. Remove the `--outFileNameData` option from plotHeatmap, since `--outFileNameMatrix` is really already doing what it should.
 2. Modify how the function is called in plotHeatmap so it saves the heatmap data, rather than the profile data.
 3. Change the documentation to note this.